### ISSUE TITLE
how use with img2img and inpaint?

### ISSUE THREAD

**Issue by loboere** [ali-vilab/FlashFace#3]
awesome work , I wanted to know if it is possible to use it with img2img and inpainting pipelines

**Comment by jshilong** [comment]
Unfortunately, we do not support img2img yet. We have a face inpainting model, but it requires additional approval from the company and is not included in the model released this time.

**Comment by mrkennypowers** [comment]
> Unfortunately, we do not support img2img yet. We have a face inpainting model, but it requires additional approval from the company and is not included in the model released this time.

Will there be a sdxl version sometime in the future?

**Comment by jshilong** [comment]
> > Unfortunately, we do not support img2img yet. We have a face inpainting model, but it requires additional approval from the company and is not included in the model released this time.
> 
> Will there be a sdxl version sometime in the future?

Since it was originally just a personal research project, we didn't utilize a significant amount of computational resources. However, if there is a higher demand for SDXL, we might consider using a larger model to achieve better results. Currently, our performance is indeed constrained by SD-1.5.

**Comment by mrkennypowers** [comment]
> > > Unfortunately, we do not support img2img yet. We have a face inpainting model, but it requires additional approval from the company and is not included in the model released this time.
> > 
> > 
> > Will there be a sdxl version sometime in the future?
> 
> Since it was originally just a personal research project, we didn't utilize a significant amount of computational resources. However, if there is a higher demand for SDXL, we might consider using a larger model to achieve better results. Currently, our performance is indeed constrained by SD-1.5.

That's understandable. Thank you for your quick and honest reply. I can't speak for anyone else, but i certainly would love to see this implmented in sdxl at some point as 1.5 is getting later on in its life cycle, but understand if that's not realistic. For now i'm struggling to get the demo working on my pc and am trying to convince others to get this ported to comfy or auto1111 as they're a bit easier to use for me. I really appreciate the work you've done and think this is a good step for idenity preservation for unique characters.

**Comment by jshilong** [comment]
> > > > Unfortunately, we do not support img2img yet. We have a face inpainting model, but it requires additional approval from the company and is not included in the model released this time.
> > > 
> > > 
> > > Will there be a sdxl version sometime in the future?
> > 
> > 
> > Since it was originally just a personal research project, we didn't utilize a significant amount of computational resources. However, if there is a higher demand for SDXL, we might consider using a larger model to achieve better results. Currently, our performance is indeed constrained by SD-1.5.
> 
> That's understandable. Thank you for your quick and honest reply. I can't speak for anyone else, but i certainly would love to see this implmented in sdxl at some point as 1.5 is getting later on in its life cycle, but understand if that's not realistic. For now i'm struggling to get the demo working on my pc and am trying to convince others to get this ported to comfy or auto1111 as they're a bit easier to use for me. I really appreciate the work you've done and think this is a good step for idenity preservation for unique characters.

This is my first work in the field of image generation in my research career. Your response has truly motivated me.

 I must admit that I am not very familiar with the ways algorithms are used and the demands of the generation community. I am also not familiar with "comfy or auto1111," including the requirement for the Windows version. In my previous research work, I rarely encountered such demands in releases, and I feel that this release is somewhat disconnected from the community. I will learn from this experience and hope to contribute useful algorithms to the community in the future. 

Thank you again for your feedback.

**Comment by 3blackbar** [comment]
yeah if this was comfyui node or auto1111 extension from the start you would get a ton more people here, now there is comfy node so more people will try this out, its a great method.
